local/bin/py/build/actions/workflows.py

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env python3
 import glob
 import json
-import os
+import re
 import yaml
 from itertools import chain
 import logging
 from pathlib import Path
 
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.WARNING)
+stability = 'dev'
```

Comment:
```suggestion
stability = 'stable'
```
Just leaving this here, so we can flip it before merging

Reply:
👍